pkg/server/init_local_environment.go

```diff
@@ -138,7 +139,7 @@ func (r *Server) applyKubernetesConnector(
 			},
 		}
 
-		if os.Getenv("KUBERNETES_SERVICE_HOST") == "" && os.Getenv("_RUNNING_INSIDE_CONTAINER_") != "" {
+		if servervars.ServingEmbeddedKubernetes.Get() {
```

Comment:
Previously, operator used the LabelEmbeddedKubernetes to determine the address of the discovered access endpoints. Looks like we can remove the label and related fields in operator since we no longer do endpoint discovery.

Reply:
updated